src/jaegertracing/utils/UDPClientTest.cpp

```diff
@@ -16,13 +16,13 @@
 
 #include "jaegertracing/net/IPAddress.h"
 #include "jaegertracing/net/Socket.h"
+
```

Comment:
Please remove newline. I think I used the include-what-you-use tool on the project, which reordered all the headers. I'd rather have consistency with style going forward.